scripts/sem_bleaching_map.py

```diff
@@ -311,137 +353,322 @@ def configure_sem_for_tile(self):
+        data4 = self.acquire_SEM([px_roi_width[0] + 2 * b_px[0], b_px[1]],
+                                 [px_center[0], px_center[1] + px_border_trans[1]])
+
+        # In order to reassemble the borders into one figure, we need to fill the ROI
+        zeros_ROI = numpy.zeros((px_roi_width[1], px_roi_width[0]), data1.dtype)
+
+        # Concatenate the data to get the final image
+        data = numpy.concatenate((data1, numpy.concatenate((data2, zeros_ROI, data3), 1), data4), 0)
+
+        # set the metadata
+        phys_roi = self.convert_roi_ratio_to_phys(roi)
+        center = ((phys_roi[0] + phys_roi[2]) / 2,
+                  (phys_roi[1] + phys_roi[3]) / 2)
+        md = {model.MD_POS: center,
+              model.MD_PIXEL_SIZE: (self.subpxs, self.subpxs),
+              model.MD_DWELL_TIME: self.dt,
+              model.MD_DESCRIPTION: "SEM border"}
+
+        return model.DataArray(data, md)
+
+    def acquire_SEM(self, resolution, translation):
+        self.ebeam.resolution.value = resolution
+        self.ebeam.translation.value = translation
+        data = self.sed.data.get()
+        data = numpy.array(data)
+        return data
+
     def bleach_spot(self, pos):
         """
         Bleach one spot, by scanning over each subpixel around that spot
         pos (0<=2 floats): X/Y position of the e-beam in SEM coordinates
-        return (ndarray): the SED acquisition during this bleaching 
+        return (ndarray): the SED acquisition during this bleaching
         """
         # position the ebeam
-        self.ebeam.translation.value = pos
-        
+        self.ebeam.translation.value = (pos[0] - self.drift[0],
+                                        pos[1] - self.drift[1])
+
         # .get() has an advantage over subscribe + unsubscribe to ensure the
         # ebeam stays at the spot (almost) just the requested dwell time
         # TODO: check in semcomedi that it really happens.
         data = self.sed.data.get()
-        
+
         if data.shape != self.tile_shape[::-1]:
             logging.error("Expected to get a SEM image of %s, but acquired %s points",
                           self.tile_shape, data.shape[::-1])
 
         return data
 
-    def get_fluo_image(self):
+    def get_fluo_image(self, lpower=None, exptime=None):
         """
         Acquire a CCD image and convert it into a intensity count
         return (DataArray): the light intensity (currently, the mean)
         """
         # TODO: see if there are better ways to measure the intensity (ex:
-        # use average over multiple frames after discarding outliers to 
+        # use average over multiple frames after discarding outliers to
         # compensate for cosmic rays)
-        
+
+        if lpower is None:
+            lpower = self.lpower
+
+        if exptime is None:
+            exptime = self.exptime
+
+        # Set exposure time and emissions
```

Comment:
Here it's written `#Set exposure time and emissions` but only exposure time is set